installing/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -119,4 +119,47 @@ If no data appears in the ACS Console:
 * Ensure that at least one secured cluster is connected to your {product-title-managed-short} instance. For more information, see the "Installing secured cluster resources on each cluster" section.
 * Examine your Sensor pod logs to ensure that the connection to your {product-title-managed-short} instance is successful.
 * In the {ocp-short} cluster, navigate to *Platform Configuration* -> *Clusters* to verify that the components are healthy and view additional operational information.
-* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
\ No newline at end of file
+* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
+
+[id="default-access-acs-console"]
+== Default access to ACS Console
+
+By default, the authentication mechanism available to users is authentication by using Red Hat Single Sign-On (SSO).
+You cannot delete or change the Red Hat SSO authentication provider.
+However, you can change the minimum access role and add additional rules, or add another identity provider.
+
+[NOTE]
+====
+To learn how authentication providers work in ACS, visit the following page: xref:../operating/manage-user-access/understanding-authentication-providers.adoc[Understanding authentication providers]
+====
+
+A dedicated OIDC client of `sso.redhat.com` is created for each ACS Console. All OIDC clients share the same `sso.redhat.com` realm.
+Claims from the token issued by `sso.redhat.com` are mapped to an ACS-issued token as follows:
+
+* `realm_access.roles` to `groups`
+* `org_id` to `rh_org_id`
+* `is_org_admin` to `rh_is_org_admin`
+* `sub` to `userid`
+
+The built-in Red Hat SSO authentication provider has the required attribute `rh_org_id` set to the organization ID assigned to account of the user who created the ACSCS instance.
+This is the ID of the organizational account the user is a part of. This can be thought of as the "tenant" the user is under and owned by.
+Only users with the same organizational account can access the ACS console by using the Red Hat SSO authentication provider.
+
+[NOTE]
+====
+To gain more control over access to your ACS Console, configure another identity provider instead of relying on the Red Hat SSO authentication provider.
+You can set up your own identity provider after provisioning by using one of the guides under `Operating -> Managing user access`.
```

Comment:
Operating -> Managing user access is not a menu item in the GUI, it's one of the documentation pages, so I think it should not be backticks? But bold, instead, because the user clicks on it to navigate.

Reply:
Here is the guidance:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference.

Use backticks for all non-GUI "system items", including:

Inline commands, operations, literal values, variables, parameters, settings, flags, environment variables, user input

System term/item, user names, unique or example names for individual API objects/resources (e.g., a pod named mypod), daemon, service, or software package

RPM packages

File names or directory paths